Dockerfile.QA

```diff
@@ -191,6 +191,12 @@ RUN if [ -d qa/L0_model_control_stress ]; then \
             cp -r qa/L0_model_control_stress/. qa/L0_model_control_stress_valgrind_massif; \
     fi
 
+# Create L0_client_memory_growth_weekly and L0_memory_growth_weekly for weekly test
```

Comment:
You shouldn't need to copy or rename the tests. We are just going to run the existing tests weekly (instead of nightly). We may make changes to the existing tests but we can to that in the existing code.